### DISCUSSION TITLE
[Release Vote] Vote for Release v0.5.1-rc2

### DISCUSSION THREAD

**Discussion by wangtaohz** [apache/amoro#2048] [Polls]
Hi Everyone,

I propose that we release the following RC as the official Amoro 0.5.1 release.

The commit ID is https://github.com/NetEase/amoro/commit/045ba44b36e21c3efb4b5f9ae022420a7924eea4

This corresponds to the tag: v0.5.1-rc2
https://github.com/NetEase/amoro/commits/v0.5.1-rc2
https://github.com/NetEase/amoro/tree/v0.5.1-rc2
Please check out, build, and test.

Please vote in the next 72 hours and add your test steps when voting.

**Comment by wangtaohz** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop3, flink-1.15.

Tested the following features:

- using derby as sysdb
- create the optimizer group (local container) and start a local optimizer
- create internal catalog (both Mixed Iceberg and Iceberg)
- spark insert overwrite/flink cdc (both Mixed Iceberg and Iceberg)
- table optimizing and result

**Comment by zhongqishang** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop2, flink-1.14.

Tested the following features:

- using MySQL as sysdb
- create the optimizer group (external container) and start a flink optimizer
- create external catalog (HMS + Iceberg format)
- insert by flink cdc
- table optimizing and result

**Comment by HuangFru** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop2, flink-1.14.

And tested the process of quick-start, works fine.

**Comment by zhoujinsong** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop3, flink-1.15.

Tested the following features:

- using derby as sysdb
- create the optimizer group (local container) and start a local optimizer
- create internal catalog (Iceberg format)
- spark insert overwrite/flink cdc
- table optimizing and result

But I noticed that the project version still be `0.5.1-SNAPSHOT`, it should be changed to `0.5.1` before releasing.

**Comment by Aireed** [comment]
+1.

I have built and deployed AMS locally with jdk8, hadoop2, flink-1.14.

Tested the following features:

- using derby as sysdb
- create the optimizer group (local container) and start a local optimizer
- create internal catalog (Iceberg format/mixed iceberg format)
- spark insert overwrite/flink cdc
- table optimizing and result

**Comment by baiyangtx** [comment]
![图片](https://github.com/NetEase/amoro/assets/7732224/08c5d011-6e08-495f-b657-f5421d7127d8)

Docs has some mistake. Here should be `FLINK_HADOOP_SHADE_VERSION`

**Reply by baiyangtx** [reply]
![图片](https://github.com/NetEase/amoro/assets/7732224/52f19621-f3c9-498c-a86f-a786a59eb68b)

`nc -l -p 9999 -k`  is better

**Reply by baiyangtx** [reply]
+1 
I have build and run quick demo locally, all is fine. 
There are some improvements in quick demo docs.

**Reply by zhoujinsong** [reply]
I will fix the errors in quick start docs before releasing.

**Reply by wangtaohz** [reply]
`nc -l -p 9999 -k` does not work on Mac OS.
```
nc -l -p 9999 -k
nc: missing port with option -l
usage: nc [-46AacCDdEFhklMnOortUuvz] [-K tc] [-b boundif] [-i interval] [-p source_port]
	  [--apple-recv-anyif] [--apple-awdl-unres]
	  [--apple-boundif ifbound]
	  [--apple-no-cellular] [--apple-no-expensive]
	  [--apple-no-flowadv] [--apple-tcp-timeout conntimo]
	  [--apple-tcp-keepalive keepidle] [--apple-tcp-keepintvl keepintvl]
	  [--apple-tcp-keepcnt keepcnt] [--apple-tclass tclass]
	  [--tcp-adp-rtimo num_probes] [--apple-initcoproc-allow]
	  [--apple-tcp-adp-wtimo num_probes]
	  [--setsockopt-later] [--apple-no-connectx]
	  [--apple-delegate-pid pid] [--apple-delegate-uuid uuid]
	  [--apple-kao] [--apple-ext-bk-idle]
	  [--apple-netsvctype svc] [---apple-nowakefromsleep]
	  [--apple-notify-ack] [--apple-sockev]
	  [--apple-tos tos] [--apple-tos-cmsg]
	  [-s source_ip_address] [-w timeout] [-X proxy_version]
	  [-x proxy_address[:port]] [hostname] [port[s]]
```

**Reply by wangtaohz** [reply]
`nc -lk 9999` also works well in docker.
<img width="357" alt="image" src="https://github.com/NetEase/amoro/assets/103108928/68c3b602-c4c1-4903-a74b-44fc5fd9b3e0">

**Comment by shidayang** [comment]
+1
I have built and deploy trino module.
All query is ok
